app/src/main/res/values/strings.xml

```diff
@@ -1221,35 +1221,35 @@
     <string name="lbbs_heading">Configure 185+ blocklists</string>
 
     <!-- backup / restore bottom sheet -->
-    <string name="brbs_title">Backup &amp; restore</string>
+    <string name="brbs_title">Backup and restoration</string>
     <string name="brbs_backup_title">Backup</string>
-    <string name="brbs_backup_desc">Backup app settings, network and dns logs</string>
+    <string name="brbs_backup_desc">Backup app settings, network and DNS logs</string>
     <string name="brbs_restore_title">Restore</string>
     <string name="brbs_restore_desc">Restore from a backup file</string>
 
     <string name="brbs_backup_dialog_title">Take backup</string>
-    <string name="brbs_backup_dialog_message">Backs up app settings, network and dns logs to a file. This will restart the VPN.</string>
+    <string name="brbs_backup_dialog_message">Backs up app settings, network and DNS logs to a file. This will restart the VPN.</string>
     <string name="brbs_backup_dialog_positive">Backup</string>
     <string name="brbs_backup_dialog_negative">Cancel</string>
 
     <string name="brbs_restore_dialog_title">Restore backup</string>
-    <string name="brbs_restore_dialog_message">Restore app settings, network and dns logs from a backup file. This will restart the app.</string>
+    <string name="brbs_restore_dialog_message">Restore app settings, network and DNS logs from a backup file. This will restart the app.</string>
     <string name="brbs_restore_dialog_positive">Restore</string>
     <string name="brbs_restore_dialog_negative">Cancel</string>
 
     <string name="brbs_backup_dialog_failure_title">Backup failed</string>
-    <string name="brbs_backup_dialog_failure_message">Could not backup. Try again?</string>
+    <string name="brbs_backup_dialog_failure_message">Could not perform backup. Try again?</string>
     <string name="brbs_backup_dialog_failure_positive">Retry Backup</string>
     <string name="brbs_backup_dialog_failure_negative">Dismiss</string>
 
-    <string name="brbs_restore_dialog_failure_title">Restore failed</string>
+    <string name="brbs_restore_dialog_failure_title">Restoration failed</string>
     <string name="brbs_restore_dialog_failure_message">Could not restore. Try again?</string>
-    <string name="brbs_restore_dialog_failure_positive">Retry Restore</string>
+    <string name="brbs_restore_dialog_failure_positive">Retry Restoration</string>
     <string name="brbs_restore_dialog_failure_negative">Dismiss</string>
 
-    <string name="brbs_restore_complete_toast">Restore completed, restarting app…</string>
+    <string name="brbs_restore_complete_toast">Restore completed. Restarting app…</string>
     <string name="brbs_backup_complete_toast">Backup completed</string>
-    <string name="brbs_restore_no_uri_toast">Restore failed; try again</string>
+    <string name="brbs_restore_no_uri_toast">Could not perform restoration. Try again.</string>
```

Comment:
`Restore failed. Try again.`?

Reply:
What is it that it is trying to restore?

Reply:
Restoring a backup of the underlying app's database and settings.